package.json

```diff
@@ -1,46 +1,52 @@
 {
-  "name": "@bandwidth/voice",
-  "version": "1.5.0",
-  "description": "Bandwidth's HTTP voice platform",
-  "main": "./lib/index.js",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/bandwidth/node-voice.git"
+  "version": "1.0.0",
```

Comment:
Should probably make this `2.0.0`?

Reply:
Oh yea for sure

Reply:
We also probably will be adding this file to the ignore list.  So we can go through it and modify things.  
Like getting ride of some of the APIMatic references.  But lets do that on a cleaner PR.